### PR TITLE
refactor: Update CODEOWNERS

### PR DESCRIPTION
**Pull request by averikitsch**
With this change, I propose we scope cloudevent-admins to the Eventarc team. This will prevent others from approving copybara PRs without a review from the Eventarc team.

### CODE REVIEW

**Comment by grayside** [comment]
@marcgovi

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -4,4 +4,5 @@
 # For syntax help see:
 # https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners#codeowners-syntax
 
-*     @googleapis/cloudevents-admins
+*           @googleapis/torus-dpe
+proto/      @googleapis/cloudevents-admins
```
Comment (marcgovi) [inline .github/CODEOWNERS:8]: @averikitsch could you also add [json/audit](https://github.com/googleapis/google-cloudevents/tree/main/json/audit) to the cloudevent-admins please?

**Review by marcgovi — APPROVED** [review]
Thanks for pinging, I completely missed this.

+1 to this change. It also aligns with the repo ownership we agreed upon ([go/cloudevent-repo-ownership](http://go/cloudevent-repo-ownership)).

I will most likely try to create a new group later on, with less permissive permissions than the cloudevents-admins and containing only Eventarc oncall team members, so everyone would be able to approve those bi-weekly copybara PRs. I will update this file once I do it.

**Review by iennae — APPROVED** [review]

File: .github/CODEOWNERS (hunk 1 above)
Comment (averikitsch) [reply .github/CODEOWNERS:9]: done

**Pull request merged**